docs/swagger.yaml

```diff
@@ -1660,6 +1660,56 @@ paths:
         in: path
         required: true
         type: string
+  /courses/v1/block_metadata/{usage_key_string}:
+    get:
+      operationId: courses_v1_block_metadata_read
+      summary: '**Use Case**'
+      description: |-
+        Returns the block metadata. Data like index_dictionary related to a
+            block should be fetched using this API, because they are too large for
+            the cache used by the course blocks/transformers API.
```

Comment:
I had to manually add this part as running `make swagger` introduces a lot of changes to this file unrelated to this MR.